### PR TITLE
High shovel

### PR DESCRIPTION
**Pull request by ghost**
[content][tested]
Somebody wanted this ages ago but nobody stepped up to code it.
Last PR of 2021...?

### CODE REVIEW

**Comment by pulserider** [comment]
Speedmerge this shit right now

File: code/game/objects/items/weapons/twohanded.dm [hunk 1]
```diff
@@ -190,6 +190,22 @@
...
+	var/obj/item/I
+	if(user.zone_sel.selecting == "l_hand")
+		I = M.get_held_item_by_index(GRASP_LEFT_HAND)
+	else if(user.zone_sel.selecting == "r_hand")
+		I = M.get_held_item_by_index(GRASP_RIGHT_HAND)
+	if(I && istype(I,src.type) && user.a_intent == I_HELP)
+		if (prob(50))
+			playsound(user, 'sound/items/trayhit1.ogg', 50, 1)
+		else
+			playsound(user, 'sound/items/trayhit2.ogg', 50, 1)
```
Comment (boy2mantwicethefam) [inline code/game/objects/items/weapons/twohanded.dm]: you could compact this into playsound(user, pick('sound/items/trayhit1.ogg', 'sound/items/trayhit2.ogg'), 50, 1)

**Comment by Exxion** [comment]
Based if you remove the changelog and just let it be a funny "secret"

**Comment by ghost** [comment]
> Based if you remove the changelog and just let it be a funny "secret"

Done.

**Comment by drawsstuff** [comment]
i was going to pester you about high spear but youve already done it. bravo

**Comment by SonixApache** [comment]
~~i don't get it~~ i don't want to get it also put it on some obscure wiki page

File: code/game/objects/items/weapons/twohanded.dm (hunk 1 above)
Comment (ghost) [reply code/game/objects/items/weapons/twohanded.dm]: Fixed.

File: code/game/objects/items/weapons/twohanded.dm [hunk 2]
```diff
@@ -190,6 +190,19 @@
...
 		user.drop_item(W, H, force_drop = 1)
 
+/obj/item/weapon/spear/attack(var/mob/living/M, var/mob/user)
+	var/obj/item/I
+	if(user.zone_sel.selecting == "l_hand")
+		I = M.get_held_item_by_index(GRASP_LEFT_HAND)
+	else if(user.zone_sel.selecting == "r_hand")
+		I = M.get_held_item_by_index(GRASP_RIGHT_HAND)
+	if(I && istype(I,src.type) && user.a_intent == I_HELP)
+		var/lastpart = ", but it feels too similar to doing it with a shovel, and isn't good."
```
Comment (Exxion) [inline code/game/objects/items/weapons/twohanded.dm]: Why is this even a var?

Comment (ghost) [reply code/game/objects/items/weapons/twohanded.dm]: Removed it.

**Pull request merged**